concourse/pipeline.yml

```diff
@@ -1,63 +1,55 @@
-##########################################################
-#
-# Groups are used to display jobs on the Web UI
-#
-##########################################################
-groups:
-- name: sagan
-  jobs:
-   - build-compass
+resource_types:
+- name: cf-zero-downtime-resource
```

Comment:
Instead of using a third-party resource, you could use the [CF resource](https://github.com/concourse/cf-resource) with the `current_app_name` set which indicates that it is a zero-downtime deploy.